ui/imports/utils/Utils.qml

```diff
@@ -762,6 +762,12 @@ QtObject {
         }
     }
 
+    // Returns fee "fake" token (that facilitates fee representation in locale) for chain.
+    function getNativeFeeTokenSymbol(chainID) {
```

Comment:
I think we're using the word `fee` in this fn name where we use `gas` in the others. Not sure which one is the most proper, but we should probably unify.

Reply:
@dlipicar updated.